app/components/HoverPreview/HoverPreviewLink.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import styled from "styled-components";
+import Img from "@shared/editor/components/Img";
+import Flex from "~/components/Flex";
+import {
+  Preview,
+  Title,
+  Description,
+  Card,
+  CardContent,
+  CARD_WIDTH,
+  THUMBNAIL_HEIGHT,
+} from "./Components";
+
+type Props = {
+  /** Link url */
+  url: string;
+  /** Title for the preview card */
+  title: string;
+  /** Url for thumbnail served by the link provider*/
+  thumbnailUrl: string;
+  /** Some description about the link provider */
+  description: string;
+};
+
+function HoverPreviewLink({ url, thumbnailUrl, title, description }: Props) {
+  return (
+    <Preview to={{ pathname: url }} target="_blank" rel="noopener noreferrer">
```

Comment:
```suggestion
    <Preview as="a" href={url} target="_blank" rel="noopener noreferrer">
```

Not tested, but this should work

Reply:
Not as it is, but with some other modifications, yes :+1: